src/content/docs/service-level-management/intro-slm.mdx

```diff
@@ -10,15 +10,22 @@ metaDescription: How to create and consume SLIs and SLOs with service level mana
 
 import slmServiceLevelsOverview from 'images/slm_screenshot-full_service-levels-overview.png'
 
-With New Relic you can define and consume service level indicators and service level objectives for your applications.
+Service levels were popularized by Google and became a best practice in the industry. Service levels complement existing monitoring by adding a perspective more focused on the business and the user experience.
+
+With New Relic, you can define and consume service level indicators (SLIs) and service level objectives (SLOs) for your applications. There are several relevant reasons to configure and follow your New Relic service levels:
```

Comment:
I wonder if we might move this into like a 'Benefits of this feature' section just because we usually try to have a very short intro and then more discrete sections, like 'benefits' or 'definition' or whatever. not that important though